internal/services/keyvault/key_vault_access_policy_resource.go

```diff
@@ -0,0 +1,372 @@
+package keyvault
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/keyvault/mgmt/2019-09-01/keyvault"
```

Comment:
could we change this to use the profile version if it exists?

Reply:
Indeed, profile `2020-09-01` exists and points to `2019-09-01` kv service. https://github.com/azure/azure-sdk-for-go/blob/v61.6.0/profiles/2020-09-01/keyvault/mgmt/keyvault/models.go#L15

done https://github.com/hashicorp/terraform-provider-azurestack/pull/151/commits/fc43f67ad8dd2773422152654c58c75fc649b9e1